sass/_typography.scss

```diff
@@ -0,0 +1,228 @@
+
+
+
+body {
+	font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+}
+
+
+p, ul, ol, dl, blockquote, hgroup, address, table, fieldset, figure, pre, img, .page-head, .media, .island, .islet, .pill, .responsive-container, .advert, .menu-block, .box, .input-group, .accordion, .video-wrap {
+	margin-bottom: 1.5rem;
+}
+
+
+a {
+	color: $linkblue;
+	line-height: normal;
+	&:hover{
+		color: $linkbluehover;
+		}
+}
+
+// MOVE TO VARIABLES..... WILL UPDATE WITH REMS .....
+ 
+h1  {
+	font-size: 30px;
+	line-height: 36px;
+	margin-bottom: 1.5rem;
+	@include media-breakpoint-up(lg) {
+		font-size: 36px;
+		line-height:43px;
+		}
+}
+
+
+h2 {
+	font-size: 24px;
+	line-height: 28.8px;
+	margin-bottom: 1.5rem;
+	@include media-breakpoint-up(lg) {
+		font-size: 30px;
+		line-height: 36px;
+		}
+}
+
+h3 {
+	font-size: 20px;
+	line-height: 24px ;
+	margin-bottom: 1.5rem;
+	@include media-breakpoint-up(lg) {
+		font-size: 24px;
+		line-height: 28.8px;
+		}
+}
+
+h4 {
+	font-size: 20px;
+	line-height: 24px ;
+	margin-bottom: 1.5rem;
+}
+
+
+
+h5 {
+	font-size: 16px;
+	line-height: 19.24px;
+	margin-bottom: 1.5rem;
+}
+
+.small-header,
+h6 {
+	font-size: 0.75em;
+	line-height: 0.9;
+	margin-bottom: 1.5em;
+}
+
+
+
+
+
+
+
+
+/*--------------------------
+PLAIN LIST
+-------------------------*/
+
+
+	ul {
+		padding-left: 75px; 
+		position: relative; 
+		list-style: none;
+	}
+	li {
+		font-size: 1em; 
+		line-height: 1.5; 
+		color: #333333;
+	@include media-breakpoint-only(md){ 
+		font-size: .75em;
+		}
+		&:before {
+	content: "\2022"; 
+	color: #00818F;
+	font-size: 1.3em;
+  	display: inline-block; 
+  	width: 1em;
+  	margin-left: -1em
+}
```

Comment:
These are going to affect some of the Drupal core or contrib plugins that we use on the site. For example, the **Administration Menu**, which is used by users when logged in, is made up of lists.

Reply:
Yep - work in progress - its too global generally at the moment - too many blue dots - all over the place.